project/config/semcommittee/config/core.entity_view_display.node.news.diff.yml

```diff
@@ -25,6 +25,7 @@ third_party_settings:
   field_group:
     group_additional_content_details:
       children:
+        - field_attachment
```

Comment:
I would guess the attachment field should be displayed on the main page. If the content type has a "Full" display mode you should use it over the "Default" view mode.

Reply:
Might even be worth checking with Darren if the News page should still have an attachment field as they should really be added through the publication page.